examples/usage.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 90,
```

Comment:
Any execution results in the jupyter notebook should be cleaned

Reply:
Fixed in next commit.